docs/api/products.rst

```diff
@@ -425,6 +428,249 @@ Example Response
+| channels       | Collection of channels to which product was assigned                        |
++----------------+-----------------------------------------------------------------------------+
+| translations   | Collection of translations (each contains slug and name in given language). |
+|                | Only the translation for default locale is required, the rest are optional  |
++----------------+-----------------------------------------------------------------------------+
+| options        | Options assigned to product                                                 |
++----------------+-----------------------------------------------------------------------------+
+| images         | Images assigned to product                                                  |
++----------------+-----------------------------------------------------------------------------+
+| attributes     | Collection of attributes connected with product (for example material)      |
++----------------+-----------------------------------------------------------------------------+
+| associations   | Collection of products associated with created product                      |
+|                | (for example accessories to this product)                                   |
++----------------+-----------------------------------------------------------------------------+
+| product_taxons | Collection of relations between product and taxons                          |
++----------------+-----------------------------------------------------------------------------+
+| main_taxon     | The main taxon to whose product is assigned                                 |
++----------------+-----------------------------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/v1/products/
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng"
+        -H “Accept: application/json”
+        -X POST
+        --data '
+            {
+                "code": "MUG_TH",
+                "main_taxon": "mugs",
+                "product_taxons": "category,mugs",
+                "channels": [
+                    "US_WEB"
+                ],
+                "attributes": [
+                     {
+                         "attribute": "mug_material",
+                         "locale_code": "en_US",
+                         "value": "concrete"
+                     }
+                 ],
+                "options": [
+                    "mug_type"
+                ],
+                 "associations": {
+                     "accessories": "f1fd2fab-c024-3192-9505-dfc8f2aef872,f1fd2fab-c024-3192-9505-dfc8f2aef872"
+                 },
+                "translations": {
+                    "en__US": {
+                        "name": "Theme Mug",
+                        "slug": "theme-mug"
+                    },
+                    "pl": {
```

Comment:
The convention in Sylius is to use `pl_PL`, we need to change it. It will be supported after #7376 is merged